css/workflow-export-dialog.styl

```diff
@@ -0,0 +1,11 @@
+.workflow-export-list
+  padding-left: 0
+
+  li
```

Comment:
I feel weird about adding styles like list-style-type to classes instead of an `li` tag. We did intend to use BEM styles for this component, but it didn't seem necessary because we were able to use styling on the semantic elements. Does BEM really prefer classes to tags?